sandbox/plugins/sandbox_filters_test.go

```diff
@@ -2,6 +2,10 @@ package plugins
 
```

Comment:
Not sure why his is part of the diff since the file didn't need to be touched but ok.